src/nodes.coffee

```diff
@@ -1263,17 +1263,15 @@ exports.ImportDeclaration = class ImportDeclaration extends ModuleDeclaration
     code
 
 exports.ExportDeclaration = class ExportDeclaration extends ModuleDeclaration
-  constructor: (@clause, @source) ->
-    super @clause, @source
-
   compileNode: (o) ->
     @checkScope o, 'export'
 
     code = []
     code.push @makeCode "#{@tab}export "
     code.push @makeCode 'default ' if @ instanceof ExportDefaultDeclaration
 
-    if @ instanceof ExportDefaultDeclaration is no and (@clause instanceof Assign or @clause instanceof Class)
+    if @ instanceof ExportDefaultDeclaration is no and
+    (@clause instanceof Assign or @clause instanceof Class)
```

Comment:
I think this looks really confusing when the line isn’t indented. I’d suggest adding three spaces at the beginning of the line (like I showed in my last comment).